### ISSUE TITLE
Hi there Viv from Heathgate

### ISSUE THREAD

**Issue by Vivharris** [ActivateMedia/heathgate#7]
Hi All
Please see below the first two issues that we would like to address sooner rather than later. We would be grateful for your input with regard to these.

1 We would like to improve the PDF brochures produced by the site. They need to resemble the ones produced by our Vebra Software. Currently they are extremely poor. Not sure how to attach one of our existing brochures to show you so will e-mail one over shortly.

2 We would like the back end of the site to work better with regard to floorplans and EPC graphs. They often end up on the rolling photo screens at the top of the page for no apparent reason.

Thanks
Viv

**Comment by gregrice** [comment]
@Vivharris- to add the PDF just drag and drop it to the text box when posting.
Attach files by dragging & dropping,  Choose Files selecting them, or pasting from the clipboard.

@therealmyth- would you please have a look at Viv's two requests- hopefully the pdf will go up soon.

**Comment by fredturmel** [comment]
No prob, will have a look on my lunchtime and in the evening today.  @Vivharris let me know when you will have uploaded the demo PDF file.

**Comment by Vivharris** [comment]
Please see attached the brochure we currently use for properties. Thanks Viv 
[Property details.pdf](https://github.com/ActivateMedia/heathgate/files/611973/Property.details.pdf)

**Comment by Vivharris** [comment]
Please see attached the current brochure style we use for properties.
[Property details.pdf](https://github.com/ActivateMedia/heathgate/files/611975/Property.details.pdf)

**Comment by Vivharris** [comment]
Hi Guys,
Hope you are well.
We would also like to be able to prevent a newly added property from appearing in the latest instructions feature window screen on the home page of the our website. This would mean that any property with poor photos can be removed from that screen but still appear on the site and be uploaded to the other portals.
Is that possible?
Have a good weekend.
Thanks,
Viv

T   020 7435 3344
M 07976 737172
E   vivharris@heathgate.co.uk<mailto:vivharris@heathgate.co.uk>
[cid:image011.png@01D20450.6B3DE940]

home<http://www.heathgate.com/> | sales<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=1&beds=0&baths=0&price_low=0&price_high=0&Itemid=2> | lettings<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=2&beds=0&baths=0&price_low=0&price_high=0&Itemid=3> | management<http://www.heathgate.com/index.php?option=com_content&view=article&id=3&Itemid=4> | hampstead<http://www.heathgate.com/index.php?option=com_content&view=article&id=6&Itemid=7> | contact us<http://www.heathgate.com/index.php?option=com_content&view=article&id=1&Itemid=10>

105 Heath Street, Hampstead, London. NW3 6SS
Tel: +44 (020) 7435 3344    Fax: +44 (020) 7433 3344    Email: property@heathgate.co.uk<mailto:property@heathgate.co.uk>    Website: www.heathgate.com<http://www.heathgate.com/>

                [cid:image008.png@01D24CC2.9D69F8E0]
        [Follow us on Twitter] <https://twitter.com/heathgateuk>                [Find us on Facebbok] <https://www.facebook.com/pages/Heathgate-Estate-Agents/118792764814982>




FOLLOW:                               LIKE:                              SEE:


© Copyright 2013 The Heathgate Group Limited. All rights reserved. Disclaimer: The information contained in this email is confidential and may be subject to legal privilege. Access to this email by anyone other than the intended recipient is unauthorised. If you are not the intended recipient, you must not use, copy, distribute or disclose this email or any part of its contents or take any action in reliance on it. If you have received this email in error, please notify us immediately by email or telephone +44 (0) 20 7435 3344. All reasonable precautions have been taken to ensure no viruses are present in this email. Heathgate is the trading name of The Heathgate Group Limited and is registered in England at 28 Church Road, Stanmore, Middlesex, HA7 4XR. Registered number 2535191. VAT No.: 544 6804 32

**Comment by therealmyth** [comment]
I improved the PDF brochure to look more like the provided example. Please let me know if you are happy with it, I've tested a few property and it looks much better now.

**Comment by therealmyth** [comment]
for the EPC and floorplan appearing on the rolling photo screens at the top of the page, would you have a current example on the site at the moment?

**Comment by Vivharris** [comment]
Not at the moment because we have to trash the property and then start again.
If you want to make it happen you can try reloading something we already have on site and it should create the issue.
If not we will send over the pictures when it happens again.
V

T   020 7435 3344
M 07976 737172
E   vivharris@heathgate.co.uk<mailto:vivharris@heathgate.co.uk>
[cid:image011.png@01D20450.6B3DE940]

home<http://www.heathgate.com/> | sales<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=1&beds=0&baths=0&price_low=0&price_high=0&Itemid=2> | lettings<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=2&beds=0&baths=0&price_low=0&price_high=0&Itemid=3> | management<http://www.heathgate.com/index.php?option=com_content&view=article&id=3&Itemid=4> | hampstead<http://www.heathgate.com/index.php?option=com_content&view=article&id=6&Itemid=7> | contact us<http://www.heathgate.com/index.php?option=com_content&view=article&id=1&Itemid=10>

105 Heath Street, Hampstead, London. NW3 6SS
Tel: +44 (020) 7435 3344    Fax: +44 (020) 7433 3344    Email: property@heathgate.co.uk<mailto:property@heathgate.co.uk>    Website: www.heathgate.com<http://www.heathgate.com/>

                [cid:image008.png@01D24EEB.286897B0]
        [Follow us on Twitter] <https://twitter.com/heathgateuk>                [Find us on Facebbok] <https://www.facebook.com/pages/Heathgate-Estate-Agents/118792764814982>




FOLLOW:                               LIKE:                              SEE:


© Copyright 2013 The Heathgate Group Limited. All rights reserved. Disclaimer: The information contained in this email is confidential and may be subject to legal privilege. Access to this email by anyone other than the intended recipient is unauthorised. If you are not the intended recipient, you must not use, copy, distribute or disclose this email or any part of its contents or take any action in reliance on it. If you have received this email in error, please notify us immediately by email or telephone +44 (0) 20 7435 3344. All reasonable precautions have been taken to ensure no viruses are present in this email. Heathgate is the trading name of The Heathgate Group Limited and is registered in England at 28 Church Road, Stanmore, Middlesex, HA7 4XR. Registered number 2535191. VAT No.: 544 6804 32

**Comment by therealmyth** [comment]
Hi Viv, 

 

Is this happening when you create a property from scratch or when you copy one?

 

f

**Comment by Vivharris** [comment]
both

Vivienne Harris

T   020 7435 3344
M 07976 737172
E   vivharris@heathgate.co.uk<mailto:vivharris@heathgate.co.uk>
[cid:image011.png@01D20450.6B3DE940]

home<http://www.heathgate.com/> | sales<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=1&beds=0&baths=0&price_low=0&price_high=0&Itemid=2> | lettings<http://www.heathgate.com/index.php?option=com_iproperty&view=list&id=0&stype=2&beds=0&baths=0&price_low=0&price_high=0&Itemid=3> | management<http://www.heathgate.com/index.php?option=com_content&view=article&id=3&Itemid=4> | hampstead<http://www.heathgate.com/index.php?option=com_content&view=article&id=6&Itemid=7> | contact us<http://www.heathgate.com/index.php?option=com_content&view=article&id=1&Itemid=10>

105 Heath Street, Hampstead, London. NW3 6SS
Tel: +44 (020) 7435 3344    Fax: +44 (020) 7433 3344    Email: property@heathgate.co.uk<mailto:property@heathgate.co.uk>    Website: www.heathgate.com<http://www.heathgate.com/>

                [cid:image008.png@01D24F07.5A3E8DA0]
        [Follow us on Twitter] <https://twitter.com/heathgateuk>                [Find us on Facebbok] <https://www.facebook.com/pages/Heathgate-Estate-Agents/118792764814982>




FOLLOW:                               LIKE:                              SEE:


© Copyright 2013 The Heathgate Group Limited. All rights reserved. Disclaimer: The information contained in this email is confidential and may be subject to legal privilege. Access to this email by anyone other than the intended recipient is unauthorised. If you are not the intended recipient, you must not use, copy, distribute or disclose this email or any part of its contents or take any action in reliance on it. If you have received this email in error, please notify us immediately by email or telephone +44 (0) 20 7435 3344. All reasonable precautions have been taken to ensure no viruses are present in this email. Heathgate is the trading name of The Heathgate Group Limited and is registered in England at 28 Church Road, Stanmore, Middlesex, HA7 4XR. Registered number 2535191. VAT No.: 544 6804 32

**Comment by therealmyth** [comment]
Hi Viv, 

I modified the section for exporting properties to include an option to allow you to prevent properties to be featured on the home page. The Export section is now labeled "Misc. settings". It still has the option for exporting the property to 3rd party portals (Include in daily export) and I added a checkbox labeled: "Do not feature this property on the frontpage." which does pretty much what it says. It's disabled by default, so if you create a property with poor photos, just thick that option to stop it to show on the homepage. It will not affect the displaying in the other sections of the website.

Regards, 

f
